src/api/app/models/staging_workflow.rb

```diff
@@ -1,4 +1,14 @@
 class StagingWorkflow < ApplicationRecord
   belongs_to :project, inverse_of: :staging
   has_many :staging_projects, class_name: 'Project', inverse_of: :staging_workflow, dependent: :nullify
+
+  has_many :staged_requests, class_name: 'BsRequest', through: :staging_projects
+
+  def unassigned_requests
+    BsRequest.by_actions_target_projects(project.id) - staged_requests - ignored_requests
+  end
+
+  def ignored_requests
+    [] # TODO: define this method
```

Comment:
Please return ``BsRequest.none`` and not an empty array because otherwise ``unassigned_requests`` will cast everything to an array and not an ActiveRecord::Result as expected.

Reply:
Done.